src/gt4py/next/program_processors/runners/dace/transformations/auto_optimize.py

```diff
@@ -66,10 +66,11 @@ def gt_auto_optimize(
         one with stride one.
     5. If requested the function will now apply loop blocking, on the dimension
         indicated by `leading_dim`.
-    6. If requested the SDFG will be transformed to GPU. For this the
+    6. The strides of temporaries are set to match the compute order..
```

Comment:
```suggestion
    6. The strides of temporaries are set to match the compute order.
```
A suggestion for a future PR, you do not need to fix it in this PR.